packages/rainbowkit/src/components/ConnectOptions/DesktopOptions.tsx

```diff
@@ -174,61 +209,149 @@ export function DesktopOptions({ onClose }: { onClose: () => void }) {
           wallet={selectedWallet}
         />
       );
-      headerLabel = hasQrCode && `Get started with ${selectedWallet.name}`;
+      headerLabel =
```

Comment:
Just confirming, we want to use the short name even when we're not in compact mode?